rust/osm2lanes/Cargo.toml

```
@@ -3,7 +3,7 @@ name = "osm2lanes"
 version = "0.1.0"
 edition = "2021"
 authors = ["Dustin Carlino <dabreegster@gmail.com>"]
```

@droogmic, I don't actually know your name or email ^^;

---

it is super duper top secret (you probably could have found it if you tried looking, but I appreciate that you didn't)

```suggestion
authors = ["Dustin Carlino <dabreegster@gmail.com>", "Michael Droogleever Fortuyn <droogmic@gmail.com>"]
```

I'm on holidays for the next 10 days, so expect a little less from me...